【启动游戏】 D:/Game Files/ZenlessZoneZero Game/ZenlessZoneZero.exe
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
-------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 689, in run
    battery,cycle=battery_cycle(COST)    #获取体力信息并计算循环次数
                  ~~~~~~~~~~~~~^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 218, in battery_cycle
    menu('stay')
    ~~~~^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 185, in menu
    pydirectinput.press('esc', )
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 242, in wrapper
    returnVal = wrappedFunction(*args, **kwargs)
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 545, in press
    upped = keyUp(k)
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 243, in wrapper
    _handlePause(funcArgs.get("_pause"))
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 232, in _handlePause
    time.sleep(PAUSE)
    ~~~~~~~~~~^^^^^^^
KeyboardInterrupt

Exception:主线程出错
按回车键退出(Enter)Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 689, in run
    battery,cycle=battery_cycle(COST)    #获取体力信息并计算循环次数
                  ~~~~~~~~~~~~~^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 218, in battery_cycle
    menu('stay')
    ~~~~^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 185, in menu
    pydirectinput.press('esc', )
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 242, in wrapper
    returnVal = wrappedFunction(*args, **kwargs)
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 545, in press
    upped = keyUp(k)
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 243, in wrapper
    _handlePause(funcArgs.get("_pause"))
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python\Lib\site-packages\pydirectinput\__init__.py", line 232, in _handlePause
    time.sleep(PAUSE)
    ~~~~~~~~~~^^^^^^^
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 708, in <module>
    run()
    ~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 705, in run
    input('Exception:主线程出错\n按回车键退出(Enter)')
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 325, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
